docs/4.x/generated/Piwik/API/Request.md

```diff
@@ -5,7 +5,7 @@ Request
 
 Dispatches API requests to the appropriate API method.
 
-The Request class is used throughout Piwik to call API methods. The difference
+okokokThe Request class is used throughout Piwik to call API methods. The difference
```

Comment:
I'm not sure why all these changes say ok in them?

Reply:
Sorry, it looks like when I generated the documentation, it did not load the latest vendor version of one component. So my debug traces were still there, and I didn't see them when pushing. You have a new documentation version now.